src/xrdsvc/SsiProvider.cc

```diff
@@ -43,7 +43,7 @@
 #include "global/ResourceUnit.h"
 #include "wconfig/WorkerConfig.h"
 #include "wpublish/ChunkInventory.h"
-#include "xrdsvc/XrdName.h"
+#include "wcomms/XrdName.h"
```

Comment:
I don't think you need `XrdName`. I'm not seeing any real use for this class and the environment `XRDNAME`. I think this mechanism is obsolete. I might be used in the very long past to allow many workers to coexist within the same MySQL database. The current integration test launches a dedicated MariaDB server for each worker. Hence the name of the worker's database is now fixed in all test or deployment environments:
```
qservw_worker
```